Resources/public/js/views/ez-subitemlistview.js

```diff
@@ -284,6 +455,16 @@ YUI.add('ez-subitemlistview', function (Y) {
              * @type Array of {Object} array containing location structs
              */
             subitems: {},
+
+            /**
+             * Boolean to check if a subitem priority is currently selected.
+             *
+             * @default false
+             * @type Boolean
+             */
+            isOnePrioritySelected: {
```

Comment:
Well I needed something to check if a priority cell was selected, but I couldn't use the class '.ez-subitem-selected-priority-cell' either because of some logic point, for example when error icon is displayed the class is removed to hide validate and cancel buttons BUT the cell should still be selected (to avoid edit icon to be displayed when hovering other cells for example). There are others example I think but I don't have them in mind. But anyway, that's why I needed something to check if a priority cell was selected, and I think adding another class again was maybe a bit too much and I thought checking would be easier with an attribute than with a class.